network/dime_message.ksy

```diff
@@ -0,0 +1,61 @@
+meta:
+  id: dime_message
+  title: DIME (Direct Internet Message Encapsulation) Message
+  license: CC0-1.0
+  endian: be
+doc: |
+  Direct Internet Message Encapsulation (DIME)
+  is an old Microsoft specification for sending and receiving
+  SOAP messages along with additional attachments,
+  like binary files, XML fragments, and even other
+  SOAP messages, using standard transport protocols like HTTP.
+doc-ref: 
+  - http://xml.coverpages.org/draft-nielsen-dime-02.txt
+  - https://docs.microsoft.com/en-us/archive/msdn-magazine/2002/december/sending-files-attachments-and-soap-messages-via-dime
+seq:
+  - id: dime
+    type: message
+types:
+  message:
+    seq:
+    - id: message
+      type: record
+      repeat: eos
+  record:
+    seq:
+    - id: version
+      type: b5
+    - id: first_record
+      type: b1
+    - id: last_record
+      type: b1
+    - id: chunck_record
+      type: b1
+    - id: type_format
+      type: b4
+    - id: reserved
+      type: b4
+    - id: options_length
+      type: u2
+    - id: id_length
+      type: u2
+    - id: type_length
+      type: u2
+    - id: data_length
+      type: u4
```

Comment:
According [to the KSY style guide](https://doc.kaitai.io/ksy_style_guide.html#attr-id), attributes that denote a byte size of some other attribute should start with `len_`:

> For fields that designate size of some particular data structure (in bytes or some other fixed units), use `len_` prefix and name of that data structure — i.e. `len_block` (length of a single `block` entry), `len_blocks` (total length of whole `blocks` array, made of `block` entries).

i.e. <del><code>options_length</code></del> <strong><code>len_options</code></strong> etc.